ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/components/properties/FontSpec.kt

```diff
@@ -29,19 +29,21 @@ private val GoogleFontsProvider: GoogleFont.Provider = GoogleFont.Provider(
     certificates = R.array.com_google_android_gms_fonts_certs,
 )
 
+private data class ResourceFontSpec(
+    val id: Int,
+    val weight: Int?,
+    val style: FontStyle?,
+)
+
 /**
  * A `FontSpec` is a more detailed version of [FontInfo]. A [FontInfo.Name] can be resolved to a generic font
  * (e.g. "sans-serif"), a font resource provided by the app, or a device font provided by the OEM.
  *
  * Determining this is relatively costly for font resources. So this abstraction allows us to perform this logic only
- * once for each one (see [determineFontSpecs]) in the validation step, before resolving the actual font where needed.
- *
- * It also allows us to defer resolving the actual font to the UI layer, as only at that time do we know the exact
- * override that's being used. We need to know this, because we need to know the [FontWeight] for which to resolve the
- * font.
```

Comment:
I think this isn't really the case anymore, considering we need to create the font families with all weights. The only thing that needs to be determined in the call site is which font weight to use for the available font family.